tsconfig.json

```diff
@@ -11,8 +11,8 @@
     "paths": {
       "@components": ["./src/components/"],
       "@icons": ["./src/icons/"],
-      "@auth": ["./src/util/auth/"],
-      "@store": ["./src/util/store"]
+      "@authentication": ["./src/util/auth/"],
+      "@storage": ["./src/util/storage/"]
```

Comment:
`store` is a svelte thing, would rather keep it the same

Reply:
Here you are referring to the generic thing, no? Which is "storage"/"saving data". 
How you implement it either using stores/etc is something else

Reply:
i guess, i never planned to use anything besides stores since it's the svelte way. It's nice to be consistent unless there is a reason not to.

Reply:
But it's not a biggie